SPECS/libtiff/libtiff.spec

```diff
@@ -1,13 +1,14 @@
 Summary:        TIFF libraries and associated utilities.
 Name:           libtiff
 Version:        4.4.0
-Release:        1%{?dist}
+Release:        2%{?dist}
 License:        libtiff
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          System Environment/Libraries
 URL:            https://gitlab.com/libtiff/libtiff
 Source0:        https://gitlab.com/libtiff/libtiff/-/archive/v%{version}/libtiff-v%{version}.tar.gz
+Patch0:         CVE-2022-2056_CVE-2022-2057_CVE-2022-2058.patch
```

Comment:
We usually do one CVE fix per line. I think our tooling might depend on the correct format of the names of the patch files.